src/backend/web/server.js

```diff
@@ -1,11 +1,30 @@
 require('../lib/config');
+const https = require('https');
+const fs = require('fs');
+
 const app = require('./app.js');
 
 const { logger } = app.get('logger');
 const HTTP_PORT = process.env.PORT || 3000;
+const HTTPS_PORT = process.env.HTTPS_PORT || 4443;
+
+const { key, cert } = (() => {
+  const certdir = fs
+    .readdirSync('/etc/letsencrypt/live', { withFileTypes: true })
```

Comment:
These paths should probably go in the `.env`.  We won't change them (often), but if we do deploy and need them updated, it's going to be a pain to find these.